src/main/scala/me/ex0ns/inlinexkcd/bot/StringHelpers.scala

```diff
@@ -0,0 +1,16 @@
+package me.ex0ns.inlinexkcd.bot
+
+/**
+  * Created by thibault on 12/12/16.
+  */
+
+object StringHelpers {
+	implicit class MarkdownString(string: String) {
+		def bold = s"*$string*"
+		def italic = s"_$string_"
```

Comment:
This should be _${string}_ to avoid capturing the underscore as part of the name